gfx/UltimateCataclysmDemo/tileset.txt

```diff
@@ -1,4 +1,4 @@
-#Ultimate Cataclysm 25 Feb 2020 release
+#Ultimate Cataclysm 0.E release 30 March 2020
 NAME: UltimateCataclysmDemo
 VIEW: Ultica Demo
```

Comment:
do we still want to call it a "demo" for 0.E?

Reply:
I'm going to do another PR to rename it, can discuss there.